moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group.h

```diff
@@ -121,8 +121,8 @@ class MoveGroup
     */
   MoveGroup(const Options &opt, const boost::shared_ptr<tf::Transformer> &tf = boost::shared_ptr<tf::Transformer>(),
             const ros::WallDuration &wait_for_servers = ros::WallDuration());
-  MOVEIT_DEPRECATED MoveGroup(const Options &opt, const boost::shared_ptr<tf::Transformer> &tf = boost::shared_ptr<tf::Transformer>(),
-                           const ros::Duration &wait_for_servers = ros::Duration());
+  MOVEIT_DEPRECATED MoveGroup(const Options &opt, const boost::shared_ptr<tf::Transformer> &tf,
+                              const ros::Duration &wait_for_servers);
```

Comment:
According to [abi compliance checker](http://ispras.linuxbase.org/index.php/ABI_compliance_checker) this is fine: [REPORT](https://rawgit.com/rhaschke/03afceeee1e30779cbf0bcd1b27f0419/raw/87f7a0cae43c5b505a61cf0f15c46864a3290985/compat_report.html)